source/_integrations/zwave_js.markdown

```diff
@@ -124,6 +124,20 @@ While your Z-Wave mesh is permanently stored on your dongle, the additional meta
 1. Put the device you want to remove in exclusion mode. Refer to its manual how this is done.
 1. The UI should confirm that the device was removed and the device and entities will be removed from Home Assistant.
 
+## Special Z-Wave entities
+
+The Z-Wave integration provides several special entities, some of which are available for every Z-Wave device, and some of which are conditional based on the device.
+
+### Entities available for every Z-Wave device
+
+1. Node status sensor - This sensor, which is disabled by default, shows the node status for a given Z-Wave device. The available node statuses are explained in the [Z-Wave JS documentation](https://zwave-js.github.io/node-zwave-js/#/api/node?id=status) and can be used in state change automations to e.g. ping a device when it is dead, or refresh values when it wakes up.
+2. Ping button - This button can be pressed to ping a device and is an alternative to the `zwave_js.ping` service.
+3. Controller/node statistics sensors - Z-Wave JS collects statistics about communications between [nodes](https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotstatistics-updatedquot) and the [controller](https://zwave-js.github.io/node-zwave-js/#/api/controller?id=quotstatistics-updatedquot) which can be used to troubleshoot RF issues in your environment. These statistics are available in the network configuration and device info panels, but they are also available as sensors which are disabled by default.
+
+### Conditional entities
+
+1. Button to manually idle notifications - Any Notification CC values on a device that have an `idle` state will get a corresponding button entity. This button entity can be used to manually idle a notification when it doesn't automatically clear on its own.
```

Comment:
```suggestion
1. Button to manually idle notifications: Any Notification CC values on a device that have an `idle` state will get a corresponding button entity. This button entity can be used to manually idle a notification when it doesn't automatically clear on its own.
```
I don't think a have a device to try this out, otherwise I would probably understand this better. What does "idling" a notification do? Hide it? clear it? stop it from appearing altogether?
What is a Notification CC value?

Reply:
thanks for the callout, at work we would call this insider baseball terminology 🙂 . The first question - idling a notification clears it. An example notification would be when a smoke detector triggers. Some devices can auto clear/idle the notification but others require a manual clear with this button.

CC = Command Class and values are individual device properties that represent sensors or actuators on the device. This may help:  https://zwave-js.github.io/node-zwave-js/#/api/overview

Open to suggestions on how to make this more clear for users

Reply:
a device can have multiple notification CC values. e.g. one for smoke, one for carbon monoxide, etc. So one device could have many of these buttons

Reply:
maybe
Any notification CC (Command Class) values on a device that have an `idle` state will get a corresponding button entity. This button entity can be used to manually idle a notification when it doesn't automatically clear on its own. A device can have multiple notification CC values. For example one for smoke and one for carbon monoxide.

Reply:
works for me, thanks!